corehq/apps/linked_domain/models.py

```diff
@@ -26,6 +26,7 @@ class DomainLink(models.Model):
     linked_domain = models.CharField(max_length=126, null=False, unique=True)
     master_domain = models.CharField(max_length=126, null=False)
     last_pull = models.DateTimeField(null=True, blank=True)
+    deleted = models.BooleanField(default=False)
```

Comment:
You may want to [modify the base queryset](https://docs.djangoproject.com/en/1.11/topics/db/managers/#modifying-a-manager-s-initial-queryset) to exclude deleted results unless you do something explicit.  For instance:
```python
DomainLink.objects.all()  # does not include deleted objects
DomainLink.deleted_objects.all()  # only includes deleted objects
```

Reply:
b961a66